lib/sdk11/components/libraries/bootloader_dfu/bootloader.c

```diff
@@ -313,10 +322,12 @@ uint32_t bootloader_init(void)
 }
 
 
-uint32_t bootloader_dfu_start(bool ota, uint32_t timeout_ms)
+uint32_t bootloader_dfu_start(bool ota, uint32_t timeout_ms, bool cancel_timeout_on_usb)
 {
     uint32_t err_code;
 
+    m_cancel_timeout_on_usb = cancel_timeout_on_usb;

```

Comment:
This is a great idea, though I think we should exclude the OTA mode from this timeout. Although we didn't fully support OTA mode, it is still doable via nRFConnect app. 

```
m_cancel_timeout_on_usb = cancel_timeout_on_usb && !ota
```